packages/mui-base/src/Tooltip/Root/useTooltipRoot.types.ts

```diff
@@ -69,40 +60,18 @@ export interface UseTooltipRootParameters {
 }
 
 export interface UseTooltipRootReturnValue {
-  /**
-   * Whether the tooltip is open.
-   */
```

Comment:
Why are JSDocs removed? They still provide value for developers. Plus, if we decide to publish the hooks, we'll have less work to do building the documentation.

Reply:
Popover doesn't have them. They're easy enough to add back and needed to be improved anyway

Reply:
> Popover doesn't have them.
TBH it doesn't look like a good reason to remove all of them from here as well.

Sure, some of them simply rephrase what's already in the prop's name, so it's fine to remove them, but for example, "The element that positioners the tooltip popup. Store in state." has additional information that could be useful for community contributors or our future selves.